docs/ref/models/querysets.txt

```diff
@@ -184,9 +184,9 @@ executed.
 Returns a new ``QuerySet`` containing objects that match the given lookup
 parameters.
 
-The lookup parameters (``**kwargs``) should be in the format described in
-`Field lookups`_ below. Multiple parameters are joined via ``AND`` in the
-underlying SQL statement.
+The zero or more lookup parameters (``**kwargs``) should be in the format
```

Comment:
`**kwargs` are generally always optional. I think phrasing it this way does more to distract from common usage. (same statement could be written for exclude(), annotate(), etc.)